.github/workflows/docs-compilation-check.yml

```diff
@@ -35,8 +35,8 @@ jobs:
 
       - name: Install Deps
         run: npm ci
-        working-directory: ./apps/reference
+        working-directory: ./apps/docs
 
       - name: Build
         run: npm run build
```

Comment:
we may not need this file anymore actually as it'll build on vercel
but if we want to keep this - need to change the build command to use turbo